### ISSUE TITLE
Add current environment and add KUBECONFIG as config location

### ISSUE THREAD

**Issue by maggieliu** [rancher/rancher#26992]
Problem:
In de default manual of kubernetes.io it is possible to extend the kubectl binary.
If i have a simple plugin named kubectl-foo in my custom PATH variable like so:

```
#!/bin/bash
echo "I am a plugin named kubectl-foo"
```

This will work

```
$ kubectl foo
I am a plugin named kubectl-foo
```
when i pass it to rancher

```
rancher kubectl foo
Error: unknown command "foo" for "kubectl"
```

**Comment by deniseschannon** [comment]
Available in Rancher CLI v2.4.4-rc1

**Comment by rancher-max** [comment]
#### Validated in CLI v2.4.4-rc1 

Followed the steps from https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/ to extend the kubectl library:
1. Create a file named `kubectl-foo` with:
```bash
#!/bin/bash

# optional argument handling
if [[ "$1" == "version" ]]
then
    echo "1.0.0"
    exit 0
fi

# optional argument handling
if [[ "$1" == "config" ]]
then
    echo "$KUBECONFIG"
    exit 0
fi

echo "I am a plugin named kubectl-foo"
``` 
2. Make it executable: `sudo chmod +x ./kubectl-foo`
3. Move it to a directory in my PATH: `sudo mv ./kubectl-foo /usr/local/bin`
4. Validate it is working as expected:
```
$ kubectl foo
I am a plugin named kubectl-foo

$ kubectl foo version
1.0.0

$ kubectl foo config
/path/to/my/kubeconfig/file
```

5. Using CLI v2.4.3, validated this is not working:
```
$ rancher login $RANCHER_SERVER_URL --token $USER_TOKEN --skip-verify

$ rancher kubectl foo
Error: unknown command "foo" for "kubectl"

Did you mean this?
	top

Run 'kubectl --help' for usage.
exit status 1
```

6. Finally, retrieved cli v2.4.4-rc1 and validated working as expected:
```
$ rancher kubectl foo
I am a plugin named kubectl-foo

$ rancher kubectl foo version
1.0.0

$ rancher kubectl foo config
/path/to/kubeconfig/used/by/rancher/cli
```
